config/v1/types_infrastructure.go

```diff
@@ -1002,6 +1002,16 @@ type PowerVSPlatformStatus struct {
 	// Note: Currently only single-zone OCP clusters are supported
 	Zone string `json:"zone"`
 
+	// resourceGroup is the resource group name for new IBMCloud resources created for a cluster.
+	// The resource group specified here will be used by cluster-image-registry-operator to set up a COS Instance in IBMCloud for the cluster registry.
+	// More about resource groups can be found here: https://cloud.ibm.com/docs/account?topic=account-rgs.
```

Comment:
Ok, few bits to update on the godoc but otherwise I think this is good
```suggestion
	// More about resource groups can be found here: https://cloud.ibm.com/docs/account?topic=account-rgs.
	// The resourceGroup must consist of only alphanumeric characters, hyphens (-), underscores (_) and spaces and has a maximum length of 40 characters.
	// When omitted ...
```
Please finish the sentence about what happens when this value isn't present

Reply:
Added this info.